extra-dev/packages/coq-smtcoq/coq-smtcoq.dev+8.12/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+maintainer: "matthieu.sozeau@inria.fr"
+homepage: "https://smtcoq.github.io/"
+dev-repo: "git+https://github.com/smtcoq/smtcoq.git"
+bug-reports: "https://github.com/smtcoq/smtcoq/issues"
+license: "CECILL-C"
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml" {>= "4.07.1" }
+  "coq" {>= "8.12~" & < "8.13~"}
+]
+tags: [ 
+  "category:Computer Science/Decision Procedures and Certified Algorithms/Decision procedures"
+  "category:Miscellaneous/Coq Extensions"
+  "keyword: smt"
+  "keyword: sat"
```

Comment:
Please use uppercase for `SAT` and `SMT`. Otherwise, it causes duplicate keywords on https://coq.inria.fr/opam/www/

Reply:
No problem, done